hs_core/templates/pages/my-resources.html

```diff
@@ -24,12 +24,12 @@ <h4 class="panel-title">
                 <div id="filter" class="facet-list panel-collapse collapse in">
                     <div class="panel-body">
                         <div class="list-group">
-                            <ul class="list-group inputs-group" data-owned-count="{{ owned|length }}"
-                            data-shared-count="{{ shared|length }}" data-added-count="{{ discovered|length }}" >
+                            <ul class="list-group inputs-group"
+                                data-shared-count="{{ shared|length }}" data-added-count="{{ discovered|length }}">
                                 <li class="list-group-item">
-                                    <span data-facet="owned" class="badge">0</span>
+                                    <span class="badge">{{ numOwned }}</span>
```

Comment:
I like this. Ideally we should populate the other ones with context variables from the back end too.